src/components/PostDetails.tsx

```diff
@@ -1,106 +1,121 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Comment, CommentData } from '../types/Comment';
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+
+type Props = {
+  showCommentLoader: boolean;
```

Comment:
The `showCommentLoader` property is not following the conventions for boolean variables. It's better to rename it to `isCommentLoading`.